test/unit/lifecycle/properties.js

```diff
@@ -218,4 +218,36 @@ describe('lifecycle/properties', function () {
     el.prop1 = '';
     expect(el.getAttribute('prop1')).to.equal('');
   });
+
+  describe('270', function () {
```

Comment:
i think this should have a more descriptive name of the issue under test. Perhaps:

`Existing properties`